### PR TITLE
Update theme-product.js

### PR DESCRIPTION
**Pull request by kivlov**
getProductJson fetch route is wrong.

### CODE REVIEW

File: packages/theme-product/theme-product.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * @param {*} url
  */
 export function getProductJson(handle) {
-  return fetch('/product/' + handle + '.js').then(function(response) {
+  return fetch('/products/' + handle + '.js').then(function(response) {
```
Comment (antoinegrant) [inline packages/theme-product/theme-product.js]: Yep, you're right. This was an oversight on our part. Thank you for your contribution.

A test would be great, if you have the capacity to do so.

Comment (kivlov) [reply packages/theme-product/theme-product.js]: Cool. I've ran a test on this locally and it works just fine. I however discovered with this repo that if you don't have the `@shopify\theme-products` module installed globally Jest test will fail. 
jest-fail

Comment (chrisberthe) [reply packages/theme-product/theme-product.js]: I feel like this issue may be related to this fix here: https://github.com/Shopify/theme-scripts/commit/9349210711bbb8b56f6af567758c4dba3c539c56.

Is it possible you may have an outdated branch that doesn't contain this fix? Perhaps you could also try removing the `node_modules` folder and reinstalling the project's dependencies.

Comment (kivlov) [reply packages/theme-product/theme-product.js]: I have had look and pulled the latest branch, it's still happening.

Comment (antoinegrant) [reply packages/theme-product/theme-product.js]: Did you run `yarn bootstrap`? This will install all lerna packages at the root.

Comment (chrisberthe) [reply packages/theme-product/theme-product.js]: That's most likely the culprit - you'll want to ensure that the `dist` folders are created in each respective package, which `yarn bootstrap` will do.

Comment (kivlov) [reply packages/theme-product/theme-product.js]: Thanks fellas, I ran the command `yarn bootstrap` and it return with a bit of errors mainly referencing that folders already exist, at the same time `yarn test` is working now too.

**Comment by shelldandy** [comment]
https://github.com/Shopify/theme-scripts/issues/94

**Comment by antoinegrant** [comment]
@kivlov, we had to move forward with a fix, so I took the liberty to push a PR that address the issue https://github.com/Shopify/theme-scripts/pull/95. 

😄 Thank you for discovering the issue and feel free to contribute again in the future.

**Pull request closed without merging**